SConstruct

```diff
@@ -272,6 +275,63 @@ if useLz4:
 
 wt_objs = [env.Object(a) for a in wtsources]
 
+#
+# wiredtiger_java.dll
+## Usage example
+## c:\Python27\python.exe c:\Python27\Scripts\scons.py --enable-java=c:/Users/paco/Downloads/swigwin-3.0.12/swig.exe,"C:/Program Files/Java/jdk1.8.0_151/"
+useJava = GetOption("lang-java")
+if useJava:
+    useJavaSplit = useJava.split(',')
+    if len(useJavaSplit) == 2:
+        swigExe = useJavaSplit[0]
+        javaPath = useJavaSplit[1]
+        # print useJavaSplit
+        # print swigExe
+        # print javaPath
+        conf.env.Append(CPPPATH=[ javaPath + '/include'])
+        conf.env.Append(CPPPATH=[ javaPath + '/include/win32'])
+
+        swigJavaFiles = ["lang/java/src/com/wiredtiger/db/AsyncOp.java",
+        "lang/java/src/com/wiredtiger/db/AsyncOpType.java",
+        "lang/java/src/com/wiredtiger/db/Connection.java",
+        "lang/java/src/com/wiredtiger/db/Cursor.java",
+        "lang/java/src/com/wiredtiger/db/Modify.java",
+        "lang/java/src/com/wiredtiger/db/SearchStatus.java",
+        "lang/java/src/com/wiredtiger/db/Session.java",
+        "lang/java/src/com/wiredtiger/db/WT_ITEM_HOLD.java",
+        "lang/java/src/com/wiredtiger/db/WT_MODIFY_LIST.java",
+        "lang/java/src/com/wiredtiger/db/wiredtiger.java",
+        "lang/java/src/com/wiredtiger/db/wiredtigerConstants.java",
+        "lang/java/src/com/wiredtiger/db/wiredtigerJNI.java"]
+
+        swigCFile = "wiredtiger_wrap.c"
+
```

Comment:
I prefer the way the section that runs SWIG for Python organizes this. Using a cloned copy of the environment (as the Python code does) is also more flexible, as we may have to turn off certain compiler warnings only for SWIG generated code.

Reply:
@ddanderson,  I moved the java section after the python section, added a check  for  comma (",") and updated some comments, also added an error message with an example of usage.
Didn't fixed the `SWIGFLAGS`  because is the first time I use Scons and I'm not sure how to do it the way you want, so please feel free to make the necessary modifications to fit your needs.